src/main/java/com/google/firebase/projectmanagement/ShaCertificateType.java

```diff
@@ -19,6 +19,8 @@
  * Enum denoting types of SHA certificates currently supported by Firebase.
  */
 public enum ShaCertificateType {
+  /** Certificate generated by SHA-1 hashing algorithm. */
   SHA_1,
```

Comment:
According to our own official Public API documentation, we are using the underscore in our constants. https://firebase.google.com/docs/projects/api/reference/rest/v1beta1/projects.androidApps.sha#ShaCertificateType

Reply:
In plain English it's normally written as SHA1 or as SHA-1 (see https://tools.ietf.org/html/rfc3174). I kind of think `SHA1` would have been better for our API, but it's not a big deal (especially since this is an enum value and not a class name).